morpheushelper/morpheushelper.py

```diff
@@ -45,8 +48,8 @@
                      /_/                                      /_/
 
 """.splitlines()
-print("\n".join(f"\033[1m\033[36m{line}\033[0m" for line in banner))
-print(f"Starting MorpheusHelper v{VERSION} ({GITHUB_LINK})\n")
+logging.info("\n".join(f"\033[1m\033[36m{line}\033[0m" for line in banner))
```

Comment:
Please use the c-style string formatting (`%s`) for lazy evaluation. also a for loop might be better.